src/open_inwoner/templates/pages/kvk/branches.html

```diff
@@ -2,36 +2,106 @@
 {% load i18n static form_tags grid_tags button_tags card_tags button_tags %}
 
 {% block content %}
-<p> {% trans "Selecteer de vestiging waarmee u wilt inloggen" %}:</p>
-{% render_grid %}
-    {% render_column start=0 span=6 %}
-        <form method="POST" id="eherkenning-branch-form">
-            {% csrf_token %}
-            {% if form.branch_number.errors %}
-                {% errors errors=form.branch_number.errors %}
-            {% endif %}
-            {% for branch in company_branches %}
-                {% render_card direction='horizontal' %}
+    <h1 class="h1" id="title">{% trans "Log in" %}</h1>
+    {% render_grid %}
+        {% render_column start=4 span=6 %}
+            <form method="POST" id="eherkenning-branch-form" class="form">
+                {% csrf_token %}
+                {% if form.branch_number.errors %}
+                    {% errors errors=form.branch_number.errors %}
+                {% endif %}
+
+                <ul class="choice-list choice-list--single">
+
+                {# if COMPANY shows NEVENVESTIGING and HOOFDVESTIGING in one list #}
+                <p><strong>{% trans "Selecteer de vestiging waarmee u wilt inloggen" %}</strong></p>
+                {% for branch in company_branches %}
                     {% with company_id=branch.vestigingsnummer %}
-                        <label class="left" for="branch-{{ company_id }}">
-                            <p>{{ branch.naam }}</p>
-                            {% if branch.straatnaam and branch.plaats %}
-                                <p>{{ branch.straatnaam }}, {{ branch.plaats}}</p>
-                            {% endif %}
-                            {% if branch.vestigingsnummer %}
-                                <p>{% trans "Vestiging" %}: {{ company_id }}</p>
-                            {% endif %}
-                        </label>
-                        <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}">
+
+                        <li class="choice-list__item">
+                            <label class="choice-list__label">
+                                <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}" class="choice-list__radio">
+                                <div class="choice-list__content">
+                                    <h2 class="choice-list__heading">{{ branch.naam }}</h2>
+                                    {% if branch.straatnaam %}
+                                    <p class="choice-list__p">{{ branch.straatnaam }}</p>
```

Comment:
It does seem there is some location info being passed, you could access it with `{{ branch.adres.binnenlandsAdres.straatnaam }}` but it doesn't have huisnummers unfortunately:

```

   {
      "adres":{
         "binnenlandsAdres":{
            "plaats":"Lollum",
            "straatnaam":"Hizzaarderlaan",
            "type":"bezoekadres"
         }
      },
      "kvkNummer":"68750110",
      "links":[
         {
            "href":"https://api.kvk.nl/test/api/v1/basisprofielen/68750110",
            "rel":"basisprofiel"
         },
         {
            "href":"https://api.kvk.nl/test/api/v1/vestigingsprofielen/000037178598",
            "rel":"vestigingsprofiel"
         }
      ],
      "naam":"Test BV Donald",
      "type":"hoofdvestiging",
      "vestigingsnummer":"000037178598"
   },
   
```

@pi-sigma if we want the full address for each vestiging we might have to do separate calls to `/v1/vestigingsprofielen/<vestigingsnummer>`?